### ISSUE TITLE
Confusion related to the training script

### ISSUE THREAD

**Issue by avijit9** [jonmun/MM-SADA-code#2]
Hi,

Thanks for releasing the code. 

Why there are two steps of training? Why not training directly from pre-trained weights for 9001 iterations?

**Comment by jonmun** [comment]
Hi,

There are 2 reasons:
1) As specified in the paper, there are 2 stages to learning the model. The first 3000 steps are for pre-training with the classification and self-supervised losses. During the final 6000 steps, the model is fine-tuned at a lower learning rate with the addition of domain discriminators.
2) Reduce the amount of computation during research. Multiple experiments share the same training strategy for the first 3000 steps (e.g. self-supervised baseline and MM-SADA), but use different strategy for fine-tuning the last 6000 steps (with or without adversarial domain alignment). Therefore a model learned for the first 3000 steps can be reused for both baselines, reducing the amount of computation.

**Comment by avijit9** [comment]
Gotcha! Thanks :)
